docs/quickstart.rst

```diff
@@ -117,35 +135,52 @@ we can evaluate metrics for subgroups within the data as below:
     >>> from fairlearn.metrics import MetricFrame
     >>> from sklearn.metrics import accuracy_score
     >>> from sklearn.tree import DecisionTreeClassifier
+    >>> from sklearn.model_selection import train_test_split
+    >>>
+    >>> X_train, X_test, y_train, y_test, \
+    A_train, A_test = train_test_split(X_ohe, y, race, random_state=123)
     >>>
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
-    >>> classifier.fit(X, y_true)
+    >>> classifier.fit(X_train, y_train)
     DecisionTreeClassifier(...)
-    >>> y_pred = classifier.predict(X)
-    >>> mf = MetricFrame(metrics=accuracy_score, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
+    >>> y_pred = (classifier.predict_proba(X_test)[:,1] >= 0.1)
+    >>> mf = MetricFrame(metrics=accuracy_score, y_true=y_test, y_pred=y_pred, \
+    sensitive_features=A_test)
     >>> mf.overall
-    0.8443...
+    0.529185173538776
     >>> mf.by_group
-    sex
-    Female    0.9251...
-    Male      0.8042...
+    race
+    AfricanAmerican    0.542303
+    Asian              0.690323
+    Caucasian          0.520261
+    Hispanic           0.595528
+    Other              0.561798
+    Unknown            0.591525
     Name: accuracy_score, dtype: float64
 
-Additionally, Fairlearn has lots of other standard metrics built-in, such as
+Note that above, because the outcome variable exhibits label imbalance, we set 
+the threshold for triggering a positive prediction lower 
+(to a probability of 0.1 or greater).
+Fairlearn has many standard metrics built-in, such as
 selection rate, i.e., the percentage of the population which have '1' as
 their label:
 
 .. doctest:: quickstart
     :options:  +NORMALIZE_WHITESPACE
 
     >>> from fairlearn.metrics import selection_rate
-    >>> sr = MetricFrame(metrics=selection_rate, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
+    >>> sr = MetricFrame(metrics=selection_rate, y_true=y_test, y_pred=y_pred, \
+    sensitive_features=A_test)
     >>> sr.overall
-    0.1638...
+    0.5124012420895405
```

Comment:
Strongly suggest ellipsis as before because it helps doctest succeed when there are minute differences in the 10th digit etc.

This applies to any number with more than 3 beyond the comma

Reply:
Done!